src/app/ui/icons/icons-list.ts

```diff
@@ -0,0 +1,53 @@
+export const iconsList = [
```

Comment:
If this is only for storybook, isn't a node function to get list from tree trivial and not require updating?

Reply:
I can look at this, I followed what the storybook example did, but that might be a better so can give it a shot.

Reply:
After looking at this, I'm not sure it is trivial to do and we might want to control the gallery shown manually? I did reorg the files a bit so there is a `docs` folder which contains the mdx file and the list to display in the docs gallery.

Reply:
Nvm, this is pretty trivial so changed.